contracts/Comptroller/Diamond/facets/PolicyFacet.sol

```diff
@@ -401,6 +434,7 @@ contract PolicyFacet is IPolicyFacet, XVSRewardsHelper {
      */
     // solhint-disable-next-line no-unused-vars
     function transferVerify(address vToken, address src, address dst, uint256 transferTokens) external {
+        ensureUserAllowed(dst);
```

Comment:
I think this is not needed, because we are already checking the permission for transferring in the `transferAllowed` check (the "pre-hook")